ui/cypress/e2e/spec.cy.ts

```diff
@@ -82,7 +82,7 @@ describe('template spec', () => {
     // Check that workspace is rendered with correct number of tables in Object Viewer
     cy.url().should('include', '/workspace');
     cy.fixture('mysql-config').then((json) => {
-      cy.get('#table-and-index-list').find('tbody tr').should('have.length', json.tableCount + 2);
+      cy.get('#schema-object-list').find('tbody tr').should('have.length', json.tableCount + 3);
     })
```

Comment:
Add a verification that the sequence node has 1 value in it , since its collapsed